index.html

```diff
@@ -74,16 +74,16 @@ <h1 class="intro_2">
         <img src="images/card_1_image_desk.svg" alt="Image for project 1 small screen"/>
```

Comment:
[ OPTIONAL ] Please add unnecessary files and folders like `.vscode` to `.gitignore`.